### PR TITLE
CON-1020: Fix Graal container scripts and add publication to Maven Central

### PR DESCRIPTION
**Pull request by bon000**
Also reverting the version back to `-1.3` to be inline with 1.3 release.

### CODE REVIEW

File: graal-sdk/build.gradle [hunk 1]
```diff
@@ -26,16 +22,31 @@ task buildGraalSDK(type: Exec) {
     }
 }
 
-////////////////////////////////////////////////////////////////////////////////////////////////////
+task sourcesJar(type: Jar) {
+    from sourceSets.main.allJava
+    classifier = 'sources'
+}
+
+task javadocJar(type: Jar) {
```
Comment (bon000) [inline graal-sdk/build.gradle]: Not needed in `graalvm` as the `packaging` type is `tar.gz` which should not require us to produce docs jar and sources jar

Comment (shamsasari) [reply graal-sdk/build.gradle]: Let's hope so!

File: graal-sdk/build.gradle [hunk 2]
```diff
@@ -26,16 +22,31 @@ task buildGraalSDK(type: Exec) {
     }
 }
 
-////////////////////////////////////////////////////////////////////////////////////////////////////
+task sourcesJar(type: Jar) {
```
Comment (shamsasari) [inline graal-sdk/build.gradle]: Are these two tasks needed? Isn't it defined by default?

Comment (bon000) [reply graal-sdk/build.gradle]: Yes we can remove them and the (empty) jars appear to be created anyway. Removing them

**Review by shamsasari — APPROVED** [review]

**Pull request merged**